opensearchapi/api_indices-get.go

```diff
@@ -37,6 +37,7 @@ func (r IndicesGetReq) GetRequest() (*http.Request, error) {
 // IndicesGetResp represents the returned struct of the get indices response
 type IndicesGetResp struct {
 	Indices map[string]struct {
+                DataStream *string `json:"data_stream"`
 		Aliases  map[string]struct{} `json:"aliases"`
 		Mappings json.RawMessage     `json:"mappings"`
 		Settings json.RawMessage     `json:"settings"`
```

Comment:
The linter complains about the spacing.
It should look something like following:
```suggestion
                DataStream *string             `json:"data_stream"`
		Aliases    map[string]struct{} `json:"aliases"`
		Mappings   json.RawMessage     `json:"mappings"`
		Settings   json.RawMessage     `json:"settings"`
```

Reply:
Sure, i will have the lint fixed. thanks for looking into this